L/libcgal_julia/build_tarballs.jl

```diff
@@ -3,11 +3,15 @@
 using BinaryBuilder
 import Pkg: PackageSpec
 
+# reminder: change the above version if restricting the supported julia versions
+julia_versions = [v"1.6.3", v"1.7.0", v"1.8.0", v"1.9.0"]
+julia_compat = join("~" .* string.(getfield.(julia_versions, :major)) .* "." .* string.(getfield.(julia_versions, :minor)), ", ")
```

Comment:
**suggestion**: alternative syntax using `@.`

```suggestion
julia_compat = join((@. "~" * string(getfield(julia_versions, :major)) * "." * string(getfield(julia_versions, :minor))), ", ")
```

or shorter
```suggestion
julia_compat = join(map(v -> "~$(v.major).$(v.minor)", julia_versions), ", ")
```

or
```suggestion
julia_compat = join(map(julia_versions) do v "~$(v.major).$(v.minor)" end), ", ")
```

**concern**: this would require more "manual" maintenance by specifying newer julia versions with which the lib is compatible.  That could mean breaking for future julia versions since the lib is not available for said versions.  Not sure if I wouldn't prefer a more liberal approach just in case the ABI we depend on doesn't change that much (e.g. `^1.6`).